### DISCUSSION TITLE
Build-only sources

### DISCUSSION THREAD

**Discussion by sciurius** [harfbuzz/harfbuzz#4702] [General] [answered]
From the docs:

> There is also amalgamated source provided with HarfBuzz which reduces whole process of building HarfBuzz to g++ src/harfbuzz.cc -fno-exceptions but there is no guarantee provided with buildability and reliability of features you get.

Is there a list of sources that are actually needed for this 'reduced building'? 

Rationale: I package the harfbuzz sources with a perl module for those systems that do not have a native harfbuzz installed, and I want to omit sources that are not really needed to reduce the package size.

**Comment by behdad** [comment, marked as answer]
Read `CONFIG.md`.
